src/servers/src/http.rs

```diff
@@ -518,6 +530,14 @@ impl HttpServer {
             routing::get(handler::health).post(handler::health),
         );
 
+        let config_router = self
+            .route_config(GreptimeOptionsConfigState {
+                greptime_config_options: self.greptime_config_options.clone().unwrap(),
+            })
+            .finish_api(&mut api);
+
+        router = router.nest(&format!("/{HTTP_API_VERSION}"), config_router);
```

Comment:
> Remember to put these administration APIs under a different path from data APIs. We don't expose these APIs in some deployment.

Do I understand it correctly that `"/{HTTP_API_VERSION}/admin"`is required here?

Reply:
If it's admin API, I don't think it should be placed under "version" path. Other admin APIs like "/health", "/status" or "/metrics" don't.